test/e2e/set/scyllacluster/scyllacluster_expose.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2022 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	scyllafixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scylla"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var _ = g.Describe("ScyllaCluster Ingress", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should create ingress objects when ingress exposeOptions are provided", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc.Spec.Datacenter.Racks[0].Members = 1
+		sc.Spec.Domains = []string{"private.nodes.scylladb.com", "public.nodes.scylladb.com"}
+		sc.Spec.Alternator = &scyllav1.AlternatorSpec{
+			Port:           4242,
```

Comment:
can this default?

Reply:
current API default value (`0`) doesn't enable alterantor :(